src/shared/create-logger.js

```diff
@@ -90,7 +90,9 @@ const getTransport = (mode: Runtime): Transport => {
              * to the express request, make sure this transport is passed to
              * that middleware so that it doesn't add its own.
              */
-            return new lw.LoggingWinston();
+            return new lw.LoggingWinston({
+                level: logLevel,
```

Comment:
Added this just to be sure we're filtering out the right things (though I verified that this was working without the need for this in non-worker-based deploys).